src/App/Resources/AppResources.resx

```diff
@@ -2464,4 +2464,7 @@ select Add TOTP to store the key safely</value>
   <data name="LoginRequestHasAlreadyExpired" xml:space="preserve">
     <value>Login request has already expired.</value>
   </data>
+  <data name="LoginAttemptFromXDoYouWantToSwitch" xml:space="preserve">
```

Comment:
It's not that long to add the last three words, please do so:
`LoginAttemptFromXDoYouWantToSwitch` => `LoginAttemptFromXDoYouWantToSwitchToThisAccount`

Reply:
Yup it is better to show it in separate lines.